src/main/java/org/fcrepo/spec/testsuite/authz/WebACLinking.java

```diff
@@ -100,6 +105,9 @@ public void conflictIfNotSupportingAclLink(final String uri) {
 
         if (resource.getStatusCode() >= 400) {
             confirmPresenceOfConstrainedByLink(resource);
+        } else {
+            assertTrue(allOf(greaterThanOrEqualTo(200), lessThan(300)).matches(resource.getStatusCode()),
```

Comment:
It may be more consistent to use https://github.com/fcrepo/Fedora-API-Test-Suite/blob/master/src/main/java/org/fcrepo/spec/testsuite/AbstractTest.java#L348

...and make that method `protected` instead of `private`.